### ISSUE TITLE
dupe class

### ISSUE THREAD

**Issue by gemal** [select2/select2#5100]
Select2 4.0.5

in select2.css
there are a dupe css class
they should be combined

not sure it's possible to combine them or not?

```
.select2-container--classic[dir="rtl"] .select2-selection--multiple .select2-selection__choice {
  float: right; }

.select2-container--classic[dir="rtl"] .select2-selection--multiple .select2-selection__choice {
  margin-left: 5px;
margin-right: auto; }
```

**Comment by alexweissman** [comment]
Again, please link to specific lines in the repo, rather than dumping code into your issue without any context.

**Comment by razh** [comment]
https://github.com/select2/select2/blob/ebf10c93db7d6d7a0d1330119d4c6f32cbd231d7/src/scss/theme/classic/_multiple.scss#L57-L64

**Comment by alexweissman** [comment]
Ah yes, that is odd.  A PR would be appreciated.

**Comment by alexweissman** [comment]
Fixed as of `4.0.6-rc.1`.
